mirror-service/src/main/resources/application.properties

```diff
@@ -10,6 +10,7 @@ quarkus.kafka-streams.application-id=dtrack-mirror-service
 quarkus.kafka-streams.application-server=localhost:8092
 quarkus.kafka-streams.topics=\
   dtrack.vulnerability.mirror.osv,\
+  dtrack.vulnerability.mirror.nvd,\
   dtrack.vulnerability
 
```

Comment:
When sending BOV records to Kafka, some records exceed the max size of ~1MB. We'll need to enable compression to make this work.

We can enable zstd compression with the following property:

```ini
kafka-streams.compression.type=zstd
```

However, the producer [evaluates the message size *before* compressing it](https://stackoverflow.com/a/48304851), so we'll also have to increase the `max.request.size` property to something higher. Increasing it to 2MB does it for me:

```ini
kafka-streams.max.request.size=2097152
```